src/examples/java/software/amazon/nio/spi/examples/Main.java

```diff
@@ -35,10 +41,10 @@ public static void main(String[] args) throws IOException {
             // proves that the correct path type is being used
             assert path.getClass().getName().contains("S3Path");
 
-            System.err.println("*** READING FROM "+path.toUri()+" ***");
+            logger.info("*** READING FROM {} ***", path.toUri());
             Files.readAllLines(path)
-                    .forEach(System.out::println);
-            System.err.println("*** FINISHED READING OBJECT ***\n");
+                .forEach(logger::info);
```

Comment:
Does the Checkstyle doc object to writing to stdout and stderr? Normally that would be good but in the `examples` package we could probably ignore that check? Not a major issue.

Reply:
Yes, it is [this part](https://github.com/awslabs/aws-java-nio-spi-for-s3/pull/311/files#diff-9abde9e2da5e93b017c69a8461ca7ba480c567029e141d598ac0fe0dcc2f2b08R323) of our config (copied from aws-sdk).

I thought it would be nice to have it in the examples also, so that it comes handy to users that already use logger